bootstrap4/templatetags/bootstrap4.py

```diff
@@ -214,52 +239,59 @@ def bootstrap_css():
 
 
 @register.simple_tag
-def bootstrap_jquery():
+def bootstrap_jquery(jquery='full'):
     """
     Return HTML for jQuery tag.
 
-    Adjust url in settings. If no url is returned, we don't want this
-    statement to return any HTML.
+    Adjust the url dict in settings. If no url is returned, we
+    don't want this statement to return any HTML.
     This is intended behavior.
 
-    Default value: ``None``
+    Default value: ``full``
 
     This value is configurable, see Settings section
 
     **Tag name**::
 
-        bootstrap_javascript
+        bootstrap_jquery
 
     **Parameters**:
 
-        :jquery: Truthy to include jQuery as well as Bootstrap
+        :jquery: falsy|slim|full (default=full)
 
     **Usage**::
 
-        {% bootstrap_javascript %}
+        {% bootstrap_jquery %}
 
     **Example**::
 
-        {% bootstrap_javascript jquery=1 %}
+        {% bootstrap_javascript jquery='slim' %}
```

Comment:
this is an an example for `bootstrap_javascript` in the docstring for `bootstrap_jquery`.

Reply:
django-bootstrap3 used these examples in the `docstrings` as the documentation is automatically built from these, right? See here: https://github.com/zostera/django-bootstrap4/blob/master/docs/templatetags.rst Thanks for the feedback. Let me know!